crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -974,6 +1137,11 @@ impl RenderMeshInstanceGpuBuilder {
             Some(render_lightmap) => u16::from(*render_lightmap.slot_index),
             None => u16::MAX,
         };
+        let lightmap_slab_index = render_lightmaps
+            .render_lightmaps
+            .get(&entity)
+            .map(|lightmap| lightmap.slab_index);
+        self.shared.lightmap_slab_index = lightmap_slab_index;
```

Comment:
Check pass ordering here. Are we sure that `render_lightmaps` is populated before this runs?

Reply:
Okay, will check. I also think that `Lightmap` needs to be added to the invalidation logic.

Reply:
Looks like we're good, as `RenderLightmaps` is populated during the extraction phase, and `RenderMeshInstanceGpuBuilder::update` is called during the collection subphase of the render phase.